### DISCUSSION TITLE
collapsed problem

### DISCUSSION THREAD

**Discussion by simon-cheung** [vuepress/core#173] [Q&A] [answered]
sidebar:
```

  {
   "text": "介绍",
   "link": "/zh-cn/readme.md"
  },
  {
   "text": "设备开发",
   "children": [...],
   "isGroup":true
   },
  {
   "text": "应用开发",
   "children": [...],
   "isGroup":true
   }
```
in the sidebar, all nodes are expaned! unlike the behavior of the sidebar in the official document.

thx.

**Comment by domnantas** [comment]
As far as I know `collapsable` is not yet implemented in v2: https://github.com/vuepress/vuepress-next/discussions/147#discussioncomment-763194

**Comment by Mister-Hope** [comment, marked as answer]
It is now.😊Enjoy it.

**Reply by domnantas** [reply]
Wonderful! Thanks!
